src/app.tsx

```diff
@@ -47,11 +47,15 @@ export interface FolderFileInfo extends FileInfo {
     to: string | null,
 }
 
+interface FilesContextType {
+    addAlert: (title: string, variant: AlertVariant, key: string) => void,
+    rootInfo: FileInfo | null,
```

Comment:
"root" sounds a bit confusing here -- this is the file information for the current directory, no? I.e. `cwdInfo` or `currentDir`, or `currentDirInfo`?